torch/csrc/jit/codegen/cuda/ir_interface_nodes.h

```diff
@@ -8,6 +8,8 @@
 
 #include <torch/csrc/jit/ir/ir.h>
 
+#include <torch/csrc/jit/codegen/cuda/scheduler/mma_utils.h>
```

Comment:
It seems odd that this header requires a scheduler header. Shouldn't `GemmTileParams` be defined in somewhere more common rather than in a scheduler file?

Reply:
Sure. I wasn't quite sure if we want to put it in type.h. I guess I'll just create a gemm_type.h for this if it sounds better.